### PR TITLE
petset: added petset scale example

### PR DESCRIPTION
**Pull request by kayrus**
Petset doc didn't contain oneliner example on how to scale petsets

### CODE REVIEW

File: docs/user-guide/petset.md [hunk 1]
```diff
@@ -252,6 +252,19 @@ You can scale a Pet Set by updating the "replicas" field. Note however that the
...
+$ kubectl get petset web
+NAME    DESIRED   CURRENT   AGE
+web     2         2         40m
+
+$ kubectl patch petset web -p '{"spec":{"replicas":3}}'
+"web" patched
+
+$ kubectl get petset nginx
+NAME    DESIRED   CURRENT   AGE
+web     3         3         40m
```
Comment (bprashanth) [inline docs/user-guide/petset.md]: Can you replace this with a `kubectl get po` and show that there are actually 3 running?

**Comment by bprashanth** [comment]
Oh, thanks, I believe I'd said something about just using kubectl edit to change the replicas but this is better. Just one nit and lgtm.

**Comment by kayrus** [comment]
@bprashanth done

**Comment by bprashanth** [comment]
LGTM thanks

**Pull request merged**